findspam.py

```diff
@@ -1157,6 +1165,10 @@ def bad_ip_for_url_hostname(s, site):
 
 def asn_for_url_host(s, site, asn_list):
     for hostname in post_hosts(s, check_tld=True):
+        if hostname in ASN_WHITELISTED_WEBSITES:
```

Comment:
As indicated in a separate comment, this should be something like

    if any(hostname == x or hostname.endswith("."+x) for x in ASN_WHITELISTED_WEBSITES):